src/modules/keyboardmanager/dll/dllmain.cpp

```diff
@@ -372,20 +361,20 @@ class KeyboardManager : public PowertoyModuleIface
         //// Remap a key to behave like a modifier instead of a toggle
         //intptr_t SingleKeyToggleToModResult = KeyboardEventHandlers::HandleSingleKeyToggleToModEvent(inputHandler, data, keyboardManagerState);
 
-        //// Handle an app-specific shortcut remapping
-        //intptr_t AppSpecificShortcutRemapResult = KeyboardEventHandlers::HandleAppSpecificShortcutRemapEvent(inputHandler, data, keyboardManagerState);
+        // Handle an app-specific shortcut remapping
+        intptr_t AppSpecificShortcutRemapResult = KeyboardEventHandlers::HandleAppSpecificShortcutRemapEvent(inputHandler, data, keyboardManagerState);
 
-        //// If an app-specific shortcut is remapped then the os-level shortcut remapping should be suppressed.
-        //if (AppSpecificShortcutRemapResult == 1)
-        //{
-        //    return 1;
-        //}
+        // If an app-specific shortcut is remapped then the os-level shortcut remapping should be suppressed.
+        if (AppSpecificShortcutRemapResult == 1)
+        {
+            return 1;
+        }
 
         // Handle an os-level shortcut remapping
         intptr_t OSLevelShortcutRemapResult = KeyboardEventHandlers::HandleOSLevelShortcutRemapEvent(inputHandler, data, keyboardManagerState);
 
         // If any of the supported types of remappings took place, then suppress the key event
-        if ((SingleKeyRemapResult + OSLevelShortcutRemapResult) > 0)
+        if ((SingleKeyRemapResult + OSLevelShortcutRemapResult + AppSpecificShortcutRemapResult) > 0)

```

Comment:
Aren't we doing a redundant check here? If `AppSpecificShortcutRemapResult` was 1, then it would never have come here?

Reply:
Yes, you're right. It looks like the same applies for `SingleKeyRemapResult` as well. I'll change this entire part of code to return  `OSLevelShortcutRemapResult`.

Reply:
Made the change